src/cloud/cdn/trouble-fastly.md

```diff
@@ -139,9 +139,12 @@ To check the response headers:
    If you have not set a static route or completed the DNS configuration for the domains on your live site, use the `--resolve` flag, which bypasses DNS name resolution.
 
    ```bash
-   curl https://<live URL> -vo /dev/null -H Fastly-Debug:1 [--resolve] <live URL hostname>:443:<live IP address>
+   curl -svo /dev/null --resolve '<your_hostname>:443:<ip address of cache node>' <https URL>
```

Comment:
For variable names, I think it is better to keep them connected with the dashes. If the line wraps, then the variable stays together instead of splitting to next line. If you're using a small browser window, this can be annoying. 

```suggestion
   curl -svo /dev/null --resolve '<your_hostname>:443:<IP-address-of-cache-node>' <https-URL>
```